packages/next/src/next-devtools/dev-overlay/components/resizer/index.tsx

```diff
@@ -26,13 +22,13 @@ export const Resizer = forwardRef(function Resizer(
         transition: 'height 250ms var(--timing-swift)',
       }}
     >
-      <div ref={ref}>{children}</div>
+      <div ref={setElement}>{children}</div>
     </div>
   )
-})
+}
 
 function useMeasureHeight(
-  ref: React.RefObject<HTMLDivElement | null>,
```

Comment:
If you put a ref here, you'll end up observing an outdated current value. What is worse is that this prevents old current values from being GCed.